webClient/src/app/editor/code-editor/monaco/monaco.config.ts

```diff
@@ -31,6 +31,205 @@ const JCL_LANG = {
   mimetypes: ['application/jcl']
 };
 
+const IEASYS_LANG = {
+  // TODO
+  id: 'IEASYS',
+  extensions: [],
+  filenamePatterns: ['IEASYS\\d\\d$'],
+  aliases: ['IEASYS'],
+  mimetypes: ['application/']
```

Comment:
@1000TurquoisePogs what should be correct mimetype here?

Reply:
I guess application/ieasys, but it doesn't really exist so we are making this new. We don't sense by mimetype currently, instead by filename pattern.

Reply:
Okay I will make the change and repush